### PR TITLE
:bug: Fix the SQL Server Agent not querying in Chinese system

### PR DESCRIPTION
**Pull request by deluxebear**
Fix the SQL Server Agent not querying in Chinese system

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@deluxebear is it possible to share one example you have connecting to a Chinese MSSQL server?

File: airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java [hunk 1]
```diff
@@ -287,7 +287,7 @@ protected void assertCdcSchemaQueryable(final JsonNode config, final JdbcDatabas
   protected void assertSqlServerAgentRunning(final JdbcDatabase database) throws SQLException {
     try {
       final List<JsonNode> queryResponse = database.unsafeQuery(connection -> {
-        final String sql = "SELECT status_desc FROM sys.dm_server_services WHERE [servicename] LIKE 'SQL Server Agent%'";
+        final String sql = "SELECT status_desc FROM sys.dm_server_services WHERE [servicename] LIKE 'SQL Server Agent%' OR  [servicename] LIKE 'SQL Server 代理%' ";
```
Comment (tuliren) [inline airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java:290]: Nitpick. There is an extra space after the `OR`.

```suggestion
        final String sql = "SELECT status_desc FROM sys.dm_server_services WHERE [servicename] LIKE 'SQL Server Agent%' OR [servicename] LIKE 'SQL Server 代理%' ";
```

**Review by tuliren — APPROVED** [review]
Interesting that SQL Server uses localized service names. This approach is not scalable since it requires a different `or` condition for every language. But I cannot find a better way either.

**Comment by deluxebear** [comment]
> @deluxebear is it possible to share one example you have connecting to a Chinese MSSQL server? Can be a screenshot or showing test output.

image

**Comment by marcosmarxm** [comment]
/publish connector=connectors/source-mssql repo=deluxebear/airbyte

**Comment by marcosmarxm** [comment]
/publish connector=connectors/source-mysql-strict-encrypt repo=deluxebear/airbyte

**Comment by marcosmarxm** [comment]
/publish connector=connectors/source-mssql-strict-encrypt repo=deluxebear/airbyte

**Review by marcosmarxm — APPROVED** [review]
thanks @deluxebear

**Pull request merged**